### PR TITLE
Update button syntax

### PR DESCRIPTION
**Pull request by jadalilleboe**
Update button syntax to follow the new standard documented here: [https://skyportal.io/docs/styling.html#buttons](https://skyportal.io/docs/styling.html#buttons)

Also added the --fix flag to the eslint precommit as I found it helpful for some linting issues. This[ stackoverflow post](https://stackoverflow.com/questions/50289536/is-it-safe-to-apply-autofix-from-eslint#:~:text=Yes%2C%20it's%20safe%2C%20because%20the,your%20JS%20issues%20%5B1%5D.) says that it should be ok

### CODE REVIEW

**Comment by jadalilleboe** [comment]
will fix failures tomorrow

**Comment by jadalilleboe** [comment]
I think this is ready for review, this test seems to be flaky:
```
FAIL: skyportal/tests/frontend/sources_and_followup_etc/test_sources.py:251 in test_classifications

        driver.scroll_to_element(classifications)
    
        del_button_xpath = "//button[starts-with(@name, 'deleteClassificationButton')]"
>       driver.click_xpath(del_button_xpath, wait_clickable=False)

skyportal/tests/frontend/sources_and_followup_etc/test_sources.py:251: 
```
But it's flaky in [another PR's run as well](https://github.com/skyportal/skyportal/runs/8232689556?check_suite_focus=true#step:15:508) so I'm thinking my code changes aren't the issue.
@guynir42 this PR is related to [this PR](https://github.com/skyportal/skyportal/pull/3107)

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ repos:
     hooks:
       - id: ESLint
         name: ESLint
-        entry: npx eslint -c .eslintrc.yaml
+        entry: npx eslint -c .eslintrc.yaml --fix
```
Comment (guynir42) [inline .pre-commit-config.yaml:35]: what does this do?

**Review by guynir42 — APPROVED** [review]
LGTM. Good to have you back, Jada! (and also, wow we have a lot of buttons...)

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jadalilleboe) [reply .pre-commit-config.yaml:35]: [https://eslint.org/docs/latest/user-guide/command-line-interface#--fix](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix)
Automatically fixes linting problems. I found it helpful when there's like 80+ linting errors that would be a pain to fix manually. I guess one concern would be if the automatic linter breaks the code in some way, but maybe we could try adding the flag for a little bit and see if there are any major issues?

Comment (guynir42) [reply .pre-commit-config.yaml:35]: Yeah that sounds fine

**Pull request merged**